Fix to comments visualization in tables and fields popovers

santilapi13 opened the pull request:
I saw issue #26 and I didn't catch what it was refering to (maybe some about bringing comments from SQL code?), but I found out a few bugs in comments display while you hover the mouse over tables and tables' fields.

- If a word in a table comment (in the SidePanel) was too long, it would get out of the popover. I fixed it by adding `wordBreak:"break-word"` to the Popover's style property.
- While hovering a field in any table, it would display all its properties except for its comment, so I added it below "Default" section, adding the word-breaking fix too.

This is not a big deal but it would be my first contribution ever, so I hope it helps.

1ilit commented:
Hello! Thank for the contribution. 

I can't reproduce the overflow bug 
![image](https://github.com/drawdb-io/drawdb/assets/96800776/fe6ca7f9-fac2-438c-9d83-002817236358)

1ilit commented on src/components/EditorCanvas/Table.jsx:
```diff
@@ -219,10 +219,19 @@ export default function Table(props) {
                 }
                 position="right"
                 showArrow
+                style={{ width: "200px", wordBreak: "break-word" }}
```
This breaks the UI. 

![image](https://github.com/drawdb-io/drawdb/assets/96800776/766d872e-27ef-43a8-8b98-5448dcbb442c)

Consider removing the width size and adding a 'max-w-[{width}px]' to the comment rather than the whole popover

1ilit commented on src/components/EditorCanvas/Table.jsx:
```diff
@@ -219,10 +219,19 @@ export default function Table(props) {
+                    <p>
+                      <strong>Comment:</strong>{" "}
+                      {e.comment === "" ? (
+                        "Not comment" 
```
Replace "Not comment" with "No comment"

santilapi13 commented:
I've just committed suggested changes, but I don't know how to make the pull request again, nor if I have to do it 😅
 
The "overflow bug" happened with long but non-existent words, for example "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa". Breaking words solves this problem.

As you said, I changed the comment div style instead of the Popover style. I gave it a max width so that it grows to a certain point, but it also keeps small if there are only a few words. Now, the UI doesn't break any more. Finally, I also changed "Not comment" to "No comment".  Here you have some pictures.

![Field1](https://github.com/drawdb-io/drawdb/assets/89311154/f6b74b02-4e23-409f-b041-6e62a2b40acf)
![Field2](https://github.com/drawdb-io/drawdb/assets/89311154/68133f2
![Field3](https://github.com/drawdb-io/drawdb/assets/89311154/6eed3a67-cfa7-4362-a6ca-9a3403bfcf2e)
b-f46f-4a14-9327-43153d5c5438)

1ilit reviewed (approved):
Looks good! Thank you
